src/tree_cmd.cpp

```diff
@@ -178,6 +179,108 @@ static void PlaceTree(TileIndex tile, uint32_t r)
 	}
 }
 
+struct BlobHarmonic {
+	int amplitude;
+	float phase;
+	int frequency;
+};
+
+/**
+ * Creates a star-shaped polygon originating from (0, 0) as defined by the given harmonics.
+ * @param radius The maximum radius of the polygon. May be smaller, but will not be larger.
+ * @param harmonics Harmonics data for the polygon.
+ * @param shape Shape to fill with points.
```

Comment:
```suggestion
 * @param[out] shape Shape to fill with points.
```

to match CreateRandomStarShapedPolygon